docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -2,11 +2,14 @@
 
 Installing Ansible on specific operating systems
 ================================================
+.. note:: These instructions are provided by their respective communities. Any bugs/issues should be filed with that community to update these instructions. Ansible maintains only the pip install instructions.
```

Comment:
I'm pretty sure there must be an empty line after the title.

Reply:
I believe you are correct, now looking at the reStructured syntax in comparison to markdown.